src/authorization-service/authorization_service/lambda_func/basic_authorizer.py

```diff
@@ -0,0 +1,46 @@
+import os
+import base64
+import traceback
+
+def handler(event, context):
+    print("auth request")
+
+    auth_header = event["authorizationToken"]
+
+    try:
+        scheme, encoded_creds = auth_header.split(' ')
+        decoded_creds = base64.b64decode(encoded_creds).decode('utf-8')
+        username, password = decoded_creds.split(':')
```

Comment:
note: in HTTP Basic Authentication, username and password are joined by a single colon `:`, not by `=`.

[RFC 7617](https://datatracker.ietf.org/doc/html/rfc7617)